indexer/services/ender/src/lib/translated-types.ts

```diff
@@ -1,7 +1,34 @@
-import { IndexerAssetPosition, IndexerPerpetualPosition, IndexerSubaccountId } from '@dydxprotocol-indexer/v4-protos';
+import { Liquidity } from '@dydxprotocol-indexer/postgres';
+import {
+  IndexerAssetPosition,
+  IndexerOrder,
+  IndexerPerpetualPosition,
+  IndexerSubaccountId, LiquidationOrderV1,
+} from '@dydxprotocol-indexer/v4-protos';
+import { Long } from '@dydxprotocol-indexer/v4-protos/build/codegen/helpers';
 
 export interface SubaccountUpdate {
   subaccountId?: IndexerSubaccountId;
   updatedPerpetualPositions: IndexerPerpetualPosition[];
   updatedAssetPositions: IndexerAssetPosition[];
 }
+
+export interface OrderFillWithLiquidity {
+  makerOrder?: IndexerOrder;
+  order?: IndexerOrder;
+  liquidationOrder?: LiquidationOrderV1;
+  /** Fill amount in base quantums. */
+  fillAmount: Long;
+  /** Maker fee in USDC quantums. */
+  makerFee: Long;
+  /**
+   * Taker fee in USDC quantums. If the taker order is a liquidation, then this
+   * represents the special liquidation fee, not the standard taker fee.
+   */
+  takerFee: Long;
+  /** Total filled of the maker order in base quantums. */
+  totalFilledMaker: Long;
+  /** Total filled of the taker order in base quantums. */
+  totalFilledTaker: Long;
+  liquidity: Liquidity;
```

Comment:
nit: Add comment stating this is the liquidity of the order in the match to process in the handler.

Reply:
done

Reply:
nit: Change the comment structure to use `/* ... */` to match the above comments.